share/wake/lib/system/remote_cache_runner.wake

```diff
@@ -350,30 +351,25 @@ def thirdBy (acceptFn: a => ThirdByGroup): (list: List a) => Triple (one: List a
 
 # Posts a completed job to the remote cache
 def postJob (rscApi: RemoteCacheApi) (job: Job) (_wakeroot: String) (hidden: String) (input: RunnerInput) (output: RunnerOutput): Result Unit Error =
-    require Pass stdout = job.getJobFailedStdoutRaw
-    require Pass stderr = job.getJobFailedStderrRaw
-
-    def allOutputs = output.getRunnerOutputOutputs
-
-    def rmapStat path = match (unsafe_stat path)
-        Fail x -> Fail x
-        Pass x -> Pass (Pair path x)
-
-    require Pass outputsStat =
-        allOutputs
-        | map rmapStat
-        | findFail
-        | addErrorContext "rsc: Failed to stat files to upload"
+    def filteredOutputs = output.getRunnerOutputOutputs
+    def cleanableOutputs = output.getRunnerOutputCleanableOutputs
+    def hiddenOutputs = subtract scmp cleanableOutputs filteredOutputs
```

Comment:
Should we do this subtract after the grouping so one of the lists is smaller?

Reply:
I don't think that helps, we have to `stat` the full union of `filteredOutputs` and `cleanableOutputs` in order to determine which are symlinks and I think that will be the majority of the time spent in the function (fwiw we were always `stat`ing the full list so it shouldn't be a time regression)